packages/api/tsconfig.json

```diff
@@ -15,6 +15,6 @@
       "*": ["node_modules/*"]
     }
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "src/app.ts", "src/controllers/eventController.ts"],
```

Comment:
There should be no need for this? `src/**/*.ts` should glob all `.ts` files recursively.

Reply:
Fixed